ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/dialogs/RestorePurchasesDialog.kt

```diff
@@ -0,0 +1,205 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter.dialogs
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+
+@Composable
+internal fun RestorePurchasesDialog(
```

Comment:
In Paywalls V2 we've started to annotate all these internal composables with `@JvmSynthetic` so they are not available in java, and the files that contain it with the `@file:JvmSynthetic` as well. This is quite annoying IMO, but it will help us keep the autocomplete clean for java users (most of this is compose, which is mostly meant for Kotlin, but we still have some "present" APIs that can be called in Java)